modules/http-helper/http_helper.go

```diff
@@ -56,7 +57,7 @@ func HttpGetE(t *testing.T, url string, tlsConfig *tls.Config) (int, string, err
 	return resp.StatusCode, strings.TrimSpace(string(body)), nil
 }
 
-// HttpGetWithValidation performs an HTTP GET on the given URL and verify that you get back the expected status code and body. If either
+// HTTPGetWithValidation performs an HTTP GET on the given URL and verify that you get back the expected status code and body. If either
```

Comment:
```suggestion
// HttpGetWithValidation performs an HTTP GET on the given URL and verify that you get back the expected status code and body. If either
```
These comments should match the function name 1:1. e.g `HttpGetWithValidation`

Reply:
Should have caught that myself by looking at the linter, sorry :wink: